src/components/Explore/utils.ts

```diff
@@ -149,12 +149,39 @@ function getDatasetIdByMetric(metric: ExploreMetric): DatasetId {
   }
 }
 
+export const formatDecimalAxis = (num: number, places = 2): string => {
+  if (num === null) {
+    return '-';
+  }
+  /**
+   * If value is between 100k and 1m, display in abbreviated form.
+   * Examples
+   * - 100,000 => 100k
+   * - 1,000,000 => 1.0m
+   * - 1,100,000 => 1.1m
+   */
+  if (num > 99999) {
+    return num > 999999
+      ? `${(num / 1000000).toLocaleString(undefined, {
+          minimumFractionDigits: 1,
+          maximumFractionDigits: 1,
+        })}m`
+      : `${(num / 1000).toLocaleString(undefined, {
+          minimumFractionDigits: 0,
+          maximumFractionDigits: 0,
+        })}k`;
+  }
```

Comment:
As our labels are currently in thousands, I think this approach is sufficient for now instead of something more robust but complicated, like the ones in [this thread](https://stackoverflow.com/questions/2685911/is-there-a-way-to-round-numbers-into-a-reader-friendly-format-e-g-1-1k). However, if our labels become more precise, we'd probably want to revisit this.

Reply:
nit: Consider simplifying a bit to make it more readable, I would recommend to:

- Pass `'en'` explicitly to `.toLocaleString` to make sure that number formatting is consistent across the site (if a user has a different language set on their computer, the thousands and decimal separators could be different)
- Use either scientific notation `1e6` or `1_000_000` to make the numbers easier to read
- Consider having a second `if` statement instead of nesting the conditionals

```ts
if (num >= 1e6) {
  return `${(num / 1e6).toLocaleString('en', {/* ... */}}`;
}

if (num >= 1e5) {
  return `${(num / 1e3).toLocaleString('en', {/* ... */}}`;
}

return `${num.toLocaleString('en', {/* ... */)}`'
```

Feel free to leave as is if you think the original version is better.